Makefile

```diff
@@ -47,7 +54,16 @@ deps_fedora:
 deps_debian:
 	sudo apt-get -y update
 	sudo apt-get -y install $(DEBS)
-endif
+
+# Download and install Armadillo.
+setup_armadillo:
+	rm -rf $(TMP_DIR)armadillo
+	mkdir $(TMP_DIR)armadillo
+	cd $(TMP_DIR)armadillo
+	curl https://ftp.fau.de/macports/distfiles/armadillo/armadillo-$(ARMA_VERSION).tar.xz \
+    	| tar -xvJ &&  cd armadillo* && \
+    	cmake . && make && sudo make install && cd ..  && rm -rf armadillo*
```

Comment:
Nice, great to have this extra support, although honestly I think it's okay to force the user to install Armadillo themselves.

Reply:
Yeah, we can remove this, then we must also add this information in quick start or some other place that user is only responsible for installing armadillo.

Reply:
Up to you; it looks like now the Makefile will automatically install all dependencies *except* Armadillo?  I don't think I fully understood the workflow before.  I didn't realize that the Makefile would use apt or brew or yum/dnf when possible, and only build Armadillo from source if that wasn't available.  Anyway, I don't see any issue re-adding it.

Let me know what you think, and then we can go ahead and merge this. :+1:

Reply:
>  Anyway, I don't see any issue re-adding it.

This sounds good, We can re-add armadillo in the `MAKEFILE`. :+1:

Reply:
Should we add proper documentation regarding this workflow in `go-quickstart`?

Reply:
Yeah, if you want to update the Go quickstart, that would be great too. :+1: Thanks!